src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -132,29 +145,15 @@ const UserSearchBoxV2: React.FC<UserSearchBoxProps> = props => {
               label: defaultUserGroupHeader!.userName,
               header: defaultUserGroupHeader!,
             }
-          : undefined
-      }
-      defaultOptions={
-        defaultValue
-          ? [
-              {
-                id: defaultValue,
-                value: defaultValue,
-                label: defaultUserGroupHeader!.userName,
-                header: defaultUserGroupHeader!,
-              },
-            ]
-          : true
+          : null
```

Comment:
Out of curiosity, why change the defaultValue to null instead of undefined in this case?

Reply:
Was trying to fix an issue, but this was not the cause. Will revert